src/openrct2/Context.cpp

```diff
@@ -311,24 +310,39 @@ namespace OpenRCT2
             return EXIT_FAILURE;
         }
 
-        IScene* GetLoadingScene() override
+        PreloaderScene* GetPreloaderScene() override
```

Comment:
Then Context.h should be corrected. PreloaderScene has some extra methods that need to be exposed.

Reply:
Ah, I think I went with this because IScene is the interface. It leads to problems if I make it more specific, e.g. comparisons like:
```cpp
if (GetContext()->GetActiveScene() == GetContext()->GetPreloaderScene())
```

If it's alright with you, I'd prefer to leave it as-is.

Reply:
I've gone for the following:
```cpp
        // NB: This takes some liberty in returning PreloaderScene* instead of IScene*.
        // PreloaderScene adds some methods to Scene, which are used internally by Context.
        PreloaderScene* GetPreloaderScene() override
```